packages/core/src/App/Containers/Layout/header/header-account-actions.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { observer, useStore } from '@deriv/stores';
+import { AccountActions } from 'App/Components/Layout/Header';
+
+type THeaderAccountActions = {
+    onClickDeposit: () => void;
+};
+
+const HeaderAccountActions = observer(({ onClickDeposit }: THeaderAccountActions) => {
+    const { client, ui, notifications } = useStore();
+    const { account_type, balance, currency, country_standpoint, is_eu, is_logged_in, is_virtual } = client;
+    const {
+        account_switcher_disabled_message,
+        disableApp,
+        enableApp,
+        is_account_switcher_disabled,
+        is_accounts_switcher_on,
+        openRealAccountSignup,
+        toggleAccountsDialog,
+    } = ui;
+    const { is_notifications_visible, notifications: notifications_array, toggleNotificationsModal } = notifications;
+
+    return (
+        <div id='dt_core_header_acc-info-container' className='acc-info__container'>
+            <AccountActions
+                acc_switcher_disabled_message={account_switcher_disabled_message}
```

Comment:
is not possible to access all of these props directly from store from `AccountActions` component rather than drilling a lot of em from here?

Reply:
account actions in still in jsx. maybe we can migrate and refactor similar components in separate scope. 👍🏼